.travis.yml

```diff
@@ -36,4 +37,6 @@ before_script:
   - npm run link-daemon ./daemon-binary
 
 script:
+  - npx ganache-cli --account="0x1f0d511e990ddbfec302e266d62542384f755f6cc6b3161b2e49a2a4e6c4be3d,100000000000000000000" > /dev/null &
```

Comment:
Can use `npm deploy-ganache &` so we don't need more than one place to maintain this.

Reply:
👍